etna/stories/models.py

```diff
@@ -44,52 +44,52 @@ class InsightsIndexPage(TeaserImageMixin, MetadataPageMixin, BasePage):
 
     def get_context(self, request):
         context = super().get_context(request)
-        insights_pages = self.get_children().public().live().specific()
-        context["insights_pages"] = insights_pages
+        stories_pages = self.get_children().public().live().specific()
+        context["stories_pages"] = stories_pages
```

Comment:
```suggestion
        context["story_pages"] = StoryPage..objects.public().live().child_of(self)
```

Reply:
This is now changed to reflect categorised name `Article`.  Also ideally is not specific to this ticket in terms of variable assignment. Have condensed the assignment but retained the original expression.
`context["article_pages"] = self.get_children().public().live().specific()`